sound/soc/soc-pcm.c

```diff
@@ -513,6 +517,10 @@ static int soc_pcm_open(struct snd_pcm_substream *substream)
 		    !component->driver->ops->open)
 			continue;
 
+		if (component->driver->module_get_upon_open)
+			if (!try_module_get(component->dev->driver->owner))
```

Comment:
maybe use the same pattern as before with an && instead of an additional if?

Reply:
@plbossart fixed